### PR TITLE
[Suyabay][Hotfix][#106017266] Only logged in users can comment

### PR DESCRIPTION
**Pull request by fokosun**
Modified comments area:  Only logged in users can comment

### CODE REVIEW

File: resources/views/app/includes/contents/main.blade.php [hunk 1]
```diff
@@ -79,34 +79,52 @@
                     <ul class="collapsible" data-collapsible="accordion">
                         <li>
                             <div class="collapsible-header" style="color:#999;padding-left:15px;">
-                                comments
+                                <b>Comments</b>
                             </div>
                             <div class="collapsible-body">
                                 <ul class="collection">
+                                @if (  Auth::check() )
```
Comment (andela-iadeniyi) [inline resources/views/app/includes/contents/main.blade.php]: The closing tag </ul> should not be inside the @else

Comment (andela-fokosun) [inline resources/views/app/includes/contents/main.blade.php]: Which of the closing tags are you referring to?

File: resources/views/app/includes/contents/main.blade.php [hunk 2]
```diff
@@ -79,34 +79,52 @@
...
                                         </span>
-                                        <span class="bubble-content" style="color:#999;margin-left:40px;">
-                                                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean ipsum tellus, hendrerit tristique ligula vitae, semper eleifend ipsum.
-                                                <div class="point"></div>
+                                        <span style="color:#999;margin-left:40px;">
+                                            Only logged in users can comment.
+                                            <div class="point"></div>
                                         </span>
                                     </li>
-                            </div>
```
Comment (andela-iadeniyi) [inline resources/views/app/includes/contents/main.blade.php]: This line down should be outside. Any reason for this?

Comment (andela-fokosun) [inline resources/views/app/includes/contents/main.blade.php]: You might wanna go over the mark up again from @if (  Auth::check() ). All the tags are in the right positions. Please go over it again.

**Pull request merged**